src/dfcx_scrapi/tools/search_util.py

```diff
@@ -17,14 +17,18 @@
 import logging
 import time
 from typing import Dict
-
+from collections.abc import Sequence
```

Comment:
@Greenford use `from typing import List` instead of Collections Sequence for type hinting